spec/latest/json-ld-api/index.html

```diff
@@ -1817,7 +1817,10 @@ <h3>Algorithm</h3>
               the keys <code>@value</code> or <code>@list</code>, set <em>result</em> to
               <code>null</code>.</li>
             <li>Otherwise, if <em>result</em> is a <a class="changed">dictionary</a> whose only
-              key is <code>@id</code>, set <em>result</em> to <code>null</code>.</li>
+              key is <code>@id</code>, set <em>result</em> to <code>null</code>.
+              <span class="changed">
+                When the <code>frame expansion</code> flag is set, a <a>dictionary</a>
+                containing only the <code>@id</code> key is retained.</span></li>
```

Comment:
Missing closing tag for `</span>`?

Reply:
I think it's okay here, but a structure check showed a couple of other errors.